docusaurus/docs/dev-docs/configurations/database.md

```diff
@@ -33,7 +33,7 @@ The `./config/database.js` (or `./config/database.ts` for TypeScript) accepts 2
 
 | Parameter                                                | Description                                                                                           | Type      | Default |
 |----------------------------------------------------------|-------------------------------------------------------------------------------------------------------|-----------|---------|
-| `client`                                                 | Database client to create the connection. `sqlite` or `postgres` or `mysql` (or `mysql2`, see [additional details](#alternative-database-driver-packages) | `String`  | -       |
+| `client`                                                 | Database client to create the connection.<br/>Accepts the following values:<ul><li>For SQLite databases:<ul><li>`sqlite3`</li><li>`vscode/sqlite3`</li><li>`sqlite-legacy`</li><li>`better-sqlite3`</li></ul></li><li>For PostgreSQL databases: `postgres`</li><li>For MySQL databases:<ul><li>`mysql`</li><li>`mysql2` (see [additional details](#alternative-database-driver-packages))</li></ul></li></ul> | `String`  | -       |
```

Comment:
```suggestion
| `client`                                                 | Database client to create the connection.<br/>Accepts the following values:<ul><li>For SQLite databases:<ul><li>`sqlite3`</li><li>`better-sqlite3`</li></ul></li><li>For PostgreSQL databases: `postgres`</li><li>For MySQL databases:<ul><li>`mysql`</li><li>`mysql2` (see [additional details](#alternative-database-driver-packages))</li></ul></li></ul> | `String`  | -       |
```
I believe the two I deleted are primarily for legacy / testing purposes and we don't want people using them anyway.